.github/workflows/test_run.yml

```diff
@@ -0,0 +1,34 @@
+name: "Test run"
+
+on: [ 'pull_request' ]
+
+jobs:
+  test_run:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+
+      - name: Checking dir
+        run: echo $(ls)
+
+      - name: Update apt
+        run: sudo apt update
+
+      - name: Install pip
+        run: sudo apt install pip
+        
+      - name: install dependencies
+        run: sudo pip install requests simplejson orionutils pyyaml
+
+      - name: install dependencies_olny_simplejson
+        run: sudo pip install simplejson
+  
+      - name: Python is running
+        run: python -c 'print("Hi")'
+
+      - name: Galaxykit help is running
+        run: python -c 'from galaxykit.command import main;main();' -u admin -p admin --help
```

Comment:
`run: galaxykit -u admin -p admin --help` 

invoking main via import won't necessarily do the same thing